src/main/java/gameboy/gamepack/racinggame/view/ResultView.java

```diff
@@ -0,0 +1,31 @@
+package gameboy.gamepack.racinggame.view;
+
+import java.util.List;
+
+public class ResultView implements TextView {
+
+    private String resultText;
+
+    public ResultView(List<Integer> carsPosition) {
+        this.resultText = getRaceResult(carsPosition);
+    }
```

Comment:
자동차 출력 관련된 클래스인가요?
맞다면 생성자에서 자동차또는 드라이버를 받아서 처리하는 게 맞을 것 같아요.

```suggestion
    public ResultView(List<Car> cars) {
        // 로직
    }
```

Reply:
TextView 인터페이스를 만드신 이유도 궁금합니다. 
답변 남겨주세요~

Reply:
지금은 입출력이 모두 텍스트로 이루어지니까 단순하게 TextView를 만들어서 display메서드의 반환값이 String으로 나가도록 강제하고 나중에 다른 입출력을 요구하는 콘솔에는 다른 인터페이스를 만들어서 display메서드의 반환 타입을 강제해야겠다고 생각해서 일단 만들었는데 지금 상황에선 딱히 필요없긴합니다

Reply:
매번 모든 객체를 String으로 만들어서 반환하는 게 좀 힘들 것 같아요
타임리프나 JSP 같은 것을 써보셨으면 알겠지만
모델타입에 대한 강제성도 없고 어떻게 출력할지는 view에다가만 선언하면 됩니다.

Reply:
TextView 제외했습니다.